corehq/apps/change_feed/tests/test_connection.py

```diff
@@ -0,0 +1,11 @@
+from django.test import SimpleTestCase, override_settings
+
+from corehq.apps.change_feed.connection import get_kafka_client
+
+
+class TestKafkaConnection(SimpleTestCase):
```

Comment:
How did you decide between `SimpleTestCase` vs `TestCase` here? I've always understood `SimpleTestCase` as something not intended to interact with external systems. While I realize this works here, because 'external systems', to Django, just means that it doesn't hit a database Django is aware of, this still feels like a test that depends on external systems, and seems like a better fit for `TestCase`

Reply:
Django knows nothing about Kafka, so `SimpleTestCase` means less overhead. It can be run without creating and setting up SQL databases.